samples/apps/logging/logging.cpp

```diff
@@ -12,6 +12,10 @@
 #include "ccf/http_query.h"
 #include "ccf/user_frontend.h"
 #include "ccf/version.h"
+// FIXME: The header below is used for a single check of the certificate and
```

Comment:
FYI, the CI will fail if it finds a `TODO` in the code (we want to rely on GitHub issues to track those items). If you're used to using `FIXME`s, maybe you could add it to the `check-todo.sh` script that is run by the CI?

Reply:
I'm deliberately using FIXMEs to avoid the TODO failures. Those are meant as warnings to the reviewers and I intend to remove them before the final merge, leaving just a comment or a fix. But I don't want CI to fail "just for this", as the `ci-checks` script stops at first failure.

Reply:
Ok, most of the FIXMEs were unnecessary and just comments for the future. The only FIXME that perhaps needs to be done now is the entropy source, which I still haven't figured out how to do that.

Reply:
Added a FIXME check on `check-todo.sh`.

Reply:
FWIW, my usual pattern for this is to add TODOs, but comment out the `check-todo.sh` call in `ci-checks.sh` so that the other useful checks are run. Then I just have to remember to uncomment the `check-todo.sh` line, which can be soft-enforced with a comment on the PR (can't merge with unresolved comments).

Reply:
Just realised the entropy source is an mbedTLS thing, so no point in keeping it for OpenSSL, which already has its own PRNG implementation which I'm guessing is more than enough for our purposes.